docs/source/api.rst

```diff
@@ -35,9 +35,8 @@ The :meth:`~libertem.contrib.daskadapter.make_dask_array` function can generate
 .. testsetup:: *
 
     from libertem import api
-    from libertem.executor.inline import InlineJobExecutor
 
-    ctx = api.Context(executor=InlineJobExecutor())
+    ctx = api.Context()
```

Comment:
Apparently this triggers an "unexpected EOF" error?

Reply:
I'm trying to use a Dask client that doesn't use multiprocessing. Let's see what CI says!